packages/editor-ui/src/stores/workflows.store.ts

```diff
@@ -164,7 +164,7 @@ export const useWorkflowsStore = defineStore(STORES.WORKFLOWS, {
 			});
 		},
 		currentWorkflowHasWebhookNode(): boolean {
-			return !!this.workflow.nodes.find((node: INodeUi) => !!node.webhookId);
+			return !!this.workflow.nodes.find((node: INodeUi) => !!node.webhookId); // includes Wait node
```

Comment:
IMO this should only include wait nodes that have a `webhookId`, and for that we should update the wait node to only generate a `webhookId` when it's actually configured to do so.

Reply:
That would be better, prolly out of scope for this PR. Looks like the webhook settings are hardcoded for all cases [here](https://github.com/n8n-io/n8n/blob/e33cc2c27c07a1d81a574674c2c9949cae6a408b/packages/nodes-base/nodes/Wait/Wait.node.ts#L46-L53) which then [causes the FE](https://github.com/n8n-io/n8n/blob/4600bb3317c6bbaaca8c7c77639522656ecf7cf8/packages/editor-ui/src/views/NodeView.vue#L1960) to add `webhookId`.